### ISSUE TITLE
Where is Confluence Storage Format fiels stored?

### ISSUE THREAD

**Issue by janjur** [Arello-Mobile/sphinx-confluence#19]
As in question. After `make html` I cannot see anything different with or without the extension enabled.

**Comment by spyoungtech** [comment]
Can you try using `make clean` first then `make json` (or `make html`) ? Then check `_build/json` (or `_build/html`) directory where you should see it in confluence storage format.

I've personally noticed, if I don't do `make clean` first, it will not actually rebuild the documents.

**Comment by janjur** [comment]
Thanks. This may be an issue to fix :) Why not build it to `_build/confluence`?

**Comment by spyoungtech** [comment]
Yeah, I believe it's a sphinx mechanic behind this. To wit, sphinx won't rebuild documents unless they are new or modified, even if your configuration changes, I think. Per [the docs](https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-e) it seems you can pass the `-E` flag to sphinx-build to avoid this.

I believe the build directory is also configurable in the sphinx config or build options. `_build/json` and `_build/html` are merely defaults as I understand it. Worth noting that the confluence-publisher project assumes these default locations, but the `config.yml` for confluence-publisher has configuration keys for the build format and location.

It may be reasonable for the extension to add a custom builder name for `confluence`.
